WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -800,10 +800,7 @@ public void onActionResult(boolean succeeded) {
         // they can be restored if the user undoes the block
         final BlockedBlogResult blockResult = ReaderBlogActions.blockBlogFromReader(post.blogId, actionListener);
         // Only pass the blogID if available. Do not track feedID
-        AnalyticsUtils.trackWithSiteId(
-                AnalyticsTracker.Stat.READER_BLOG_BLOCKED,
-                mCurrentBlogId != 0 ? mCurrentBlogId : null
-        );
+        AnalyticsUtils.trackWithSiteId(AnalyticsTracker.Stat.READER_BLOG_BLOCKED, mCurrentBlogId);
```

Comment:
This resolves the issue but I'm wondering if we should be passing `post.blogId` here instead, so we track the ID of the site being blocked? I can open a separate issue for that if you like.